src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxconverter.py

```diff
@@ -11,7 +10,10 @@
 
 
 class SpikeGLXConverterPipe(ConverterPipe):
-    """Primary conversion class for handling multiple SpikeGLX data streams."""
+    """
+    The simplest, easiest to use class for converting all SpikeGLX data in a folder. Primary conversion class for
+    handling multiple SpikeGLX data streams.
```

Comment:
```suggestion
    The simplest, easiest to use class for converting all SpikeGLX data in a folder.

    Primary conversion class for handling multiple SpikeGLX data streams.
```

To avoid a future D205+D400 error when we eventually add pydocstyle